admin-center/analytics/monthly-reports.md

```diff
@@ -37,6 +37,8 @@ View a detailed report of the number of times searchers have input specific term
 
 Read [Understanding Your Users' Needs By Analyzing Search Terms](https://www.digitalgov.gov/2013/10/24/understanding-your-users-needs-by-analyzing-search-terms/) for tips on how to create a semi-automated report for analyzing the data in this CSV file on a regular basis.
```

Comment:
Just noticed - is this link still current?

Reply:
I think so - it still seems helpful!

Reply:
Ok, can you please update the domain so we're not relying on the redirect anymore? many thanks.